frontend/Dockerfile.vars

```diff
@@ -1,31 +0,0 @@
-# BUILD CLIENT
-FROM node:14-alpine as build-stage
-
-WORKDIR /app/packages/client
-
-#ENVS FOR DEVELOP FRONTEND
-
-ENV REACT_APP_BACK_END_SERVER_API "https://cast-REPLACE_NAMESPACE-backend.brud.dev"
-ENV REACT_APP_IPFS_GATEWAY "https://dappercollectives.mypinata.cloud/ipfs"
-ENV REACT_APP_FLOW_ENV "emulator"
-ENV FLOW_DEVWALLET "true"
-ENV REACT_APP_EMULATOR_API "https://develop-emulator.brud.dev:8080"
-ENV REACT_APP_EMULATOR_WALLET_DISCOVERY "https://REPLACE_NAMESPACE-wallet.brud.dev/fcl/authn"
-
-COPY ./packages/client/package*.json ./
-
-RUN npm install
-COPY ./packages/client/ ./
-COPY ./packages/shared-components/ ./
-
-ENV NODE_ENV=production
-
-RUN npm run build
-
-# DEPLOY CLIENT
-FROM nginx:latest as production-stage
-WORKDIR /app
-COPY --from=build-stage /app/packages/client/build /app
-COPY ./deploy/nginx.conf /etc/nginx/nginx.conf
-EXPOSE 80
-RUN ls -la /app
```

Comment:
@fersan1985 I don't see a reason why not deleting this....is it ok?

Reply:
Yea - I was looking at this and it seemed outdated and missing things from the other scripts. I deleted to avoid confusion. However, @fersan1985 can confirm if this file isn't used anywhere.